bin/helm-deploy

```diff
@@ -67,12 +67,22 @@ helm_upgrade() {
 }
 
 helm_template() {
-  helm template "${CHART_PATH}" \
+  if [ "${REMOTE_CHART}" == "true" ]; then
+    helm fetch --untar --untardir ./rok8s-helm-tmp "${CHART_PATH}"
```

Comment:
Can we use the ROK8S_TMP (and maybe a subdirectory of it) for this instead of ./ ?

Reply:
That's definitely better, thanks